Index: lib/screens/splashScreen.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/screens/splashScreen.dart	(revision 4e6564c2e2dbfd91b642592abcbd578163a7b13f)
+++ lib/screens/splashScreen.dart	(date 1619420596921)
@@ -7,6 +7,53 @@
 class _SplashScreenState extends State<SplashScreen> {
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      body: Stack(
+        fit: StackFit.expand,
+        children: <Widget>[
+          Container(
+            decoration: BoxDecoration(color: Color(0xfff85f6a)),
+          ),
+          Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              Expanded(
+                flex: 2,
+                child: Container(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      CircleAvatar(
+                        radius: 50,
+                        backgroundColor: Colors.blue,
+                        child: Icon(Icons.title,size: 60,color: Color(0xff1e2a78),),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.only(top: 30),
+                      ),
+                      Text("TO DO",style: TextStyle(fontFamily:'Regular' , fontSize: 25,color: Color(0xfffc5c9c) ),)
+                    ],
+                  ),
+                ),
+              ),
+              Expanded(
+                flex: 1,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    CircularProgressIndicator(
+                      valueColor: AlwaysStoppedAnimation<Color>(Colors.pink) ,
+                      backgroundColor: Color(0xff1e2a78),
+                    ),
+                    Padding(padding: EdgeInsets.only(top: 20)),
+                    Text("Manage Your Tasks Here",style: TextStyle(fontFamily:'Regular' , fontSize: 20),)
+                  ],
+                ),
+              )
+            ],
+          )
+        ],
+      ),
+    );
   }
 }
Index: lib/main.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/main.dart	(revision 4e6564c2e2dbfd91b642592abcbd578163a7b13f)
+++ lib/main.dart	(date 1619419918702)
@@ -1,3 +1,4 @@
+import 'package:chfrestaurant/screens/splashScreen.dart';
 import 'package:flutter/material.dart';
 
 void main() {
@@ -12,7 +13,9 @@
 class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return MaterialApp(
+      home: SplashScreen(),
+    );
   }
 }
 
